src/rewriter/basic_rewrite_rcons.cpp

```diff
@@ -63,6 +64,30 @@ bool BasicRewriteRCons::prove(
   return false;
 }
 
+bool BasicRewriteRCons::postProve(
+    CDProof* cdp, Node a, Node b, theory::TheoryId tid, MethodId mid)
+{
+  Node eq = a.eqNode(b);
+
+#define TRY_THEORY_REWRITE(id)                                          \
+  if (tryRule(cdp,                                                      \
+              eq,                                                       \
+              ProofRule::THEORY_REWRITE,                                \
+              {mkRewriteRuleNode(ProofRewriteRule::id), eq[0]}))        \
+  {                                                                     \
+    Trace("trewrite-rcons") << "Reconstruct " << eq << " (from " << tid \
+                            << ", " << mid << ")" << std::endl;         \
+    return true;                                                        \
+  }                                                                     \
+  /* end of macro */
```

Comment:
```suggestion
#define TRY_THEORY_REWRITE(id)                                          \
  if (tryRule(cdp,                                                      \
              eq,                                                       \
              ProofRule::THEORY_REWRITE,                                \
              {mkRewriteRuleNode(ProofRewriteRule::id), a}))            \
  {                                                                     \
    Trace("trewrite-rcons") << "Reconstruct " << eq << " (from " << tid \
                            << ", " << mid << ")" << std::endl;         \
    return true;                                                        \
  }                                                                     \
  /* end of macro */
```
nit ((feel free to ignore): it would be cleaner if the macro is defined outside the method.